packages/plugin-express/src/express.js

```diff
@@ -55,7 +55,11 @@ module.exports = {
       next(err)
     }
 
-    return { requestHandler, errorHandler }
+    const runInContext = (req, res, next) => {
+      client._clientContext.run(req.bugsnag, next)
+    }
```

Comment:
This is a new express middleware that can be used to restore async local storage context if it is lost. See E2E tes showing usage